src/sentry/dynamic_sampling/tasks/custom_rule_notifications.py

```diff
@@ -82,18 +82,24 @@ def get_num_samples(rule: CustomDynamicSamplingRule) -> int:
         # org rule get all projects for org
         projects = rule.organization.project_set.filter(status=ObjectStatus.ACTIVE)
 
-    params: dict[str, Any] = {
+    project_id = []
+    project_objects = []
```

Comment:
Isn't this list of project objects redundant?

Reply:
Hmm if you're asking why i switched to a for loop over a list comprehension was that i'm worried about hitting the queryset twice
If you're asking why not just pass projects, its cause the type is a list[project] not queryset

Reply:
I see, not big deal so feel free to disregard.